src/vector/platform/PWAPlatform.ts

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2020 New Vector Ltd
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import WebPlatform from "./WebPlatform";
+
+export default class PWAPlatform extends WebPlatform {
+    setNotificationCount(count: number) {
+        if (!navigator.setAppBadge) return super.setNotificationCount(count);
```

Comment:
Thanks for having a look at this.
Maybe we can do this straight in the web platform? The chrome post says this should be used independent of the installed state of the app...

Reply:
https://web.dev/badging-api/:
> While the Badging API in Chrome requires an installed app, you shouldn't make calls to the Badging API dependent on the install state. Just call the API when it exists, as other browsers may show the badge in other places. If it works, it works. If not, it simply doesn't.

Hmm, maybe not then, as we can't detect whether it will work, and we don't want to do the favicon fallback if not needed I imagine. Wdyt?

Reply:
> and we don't want to do the favicon fallback if not needed I imagine.

that's my thinking exactly as some obscure browser might use the favicon for the PWA & also apply this badge atop of it which would result in an ugly icon with two badges

Reply:
makes sense